x-pack/test/reporting_api_integration/reporting_and_security/generate_csv_discover.ts

```diff
@@ -6,73 +6,70 @@
  */
 
 import expect from '@kbn/expect';
-import supertest from 'supertest';
-import { JOB_PARAMS_RISON_CSV_DEPRECATED } from '../services/fixtures';
+import { SearchSourceFields } from 'src/plugins/data/common';
+import { ReportApiJSON } from '../../../plugins/reporting/common/types';
 import { FtrProviderContext } from '../ftr_provider_context';
 
 // eslint-disable-next-line import/no-default-export
 export default function ({ getService }: FtrProviderContext) {
-  const esArchiver = getService('esArchiver');
-  const supertestSvc = getService('supertest');
   const reportingAPI = getService('reportingAPI');
 
-  const generateAPI = {
-    getCsvFromParamsInPayload: async (jobParams: object = {}) => {
-      return await supertestSvc
-        .post(`/api/reporting/generate/csv`)
-        .set('kbn-xsrf', 'xxx')
-        .send(jobParams);
-    },
-    getCsvFromParamsInQueryString: async (jobParams: string = '') => {
-      return await supertestSvc
-        .post(`/api/reporting/generate/csv?jobParams=${encodeURIComponent(jobParams)}`)
-        .set('kbn-xsrf', 'xxx');
-    },
-  };
+  describe('Generate CSV from SearchSource', () => {
+    it(`exported CSV file matches snapshot`, async () => {
+      await reportingAPI.initEcommerce();
 
-  describe('Generation from Job Params', () => {
```

Comment:
All of this removed code used the "legacy" export type, and was moved to the new test file: `x-pack/test/reporting_api_integration/reporting_and_security/generate_csv_discover_legacy.ts`